src/world.ts

```diff
@@ -1,15 +1,30 @@
-import 'source-map-support/register';
+import axios from 'axios';
+import { getConfigurations } from './lib/collect-ssm-params';
 
 // AWSLambda.Handler is provides very generic typing
 // for handler functions. Specific argument and output
 // types can be supplied using generic arguments
 // e.g. AWSLambda.Handler<string, object>, and/or using
 // event-specific handler types e.g. AWSLambda.S3Handler
 export const handler: AWSLambda.Handler = async (event, context) => {
```

Comment:
Interesting that GitHub is throwing errors for this - that's typescript complaining I assume? We should probably check that in the main template.

Reply:
I got this complaining by both typescript & eslint.

`'context' is defined but never used.eslint[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)
'context' is declared but its value is never read.ts(6133)`